src/rules/__tests__/no-async-wrapper-for-expected-promise.test.ts

```diff
@@ -0,0 +1,141 @@
+import dedent from 'dedent';
+import rule from '../no-async-wrapper-for-expected-promise';
+import { FlatCompatRuleTester as RuleTester, espreeParser } from './test-utils';
+
+const ruleTester = new RuleTester({
+  parser: espreeParser,
+  parserOptions: {
+    ecmaVersion: 2017,
+  },
+});
+
+ruleTester.run('no-async-wrapper-for-expected-promise', rule, {
+  valid: [
+    'expect.hasAssertions()',
+    dedent`
+        it('pass', async () => {
+            expect();
+        })
+        `,
+    dedent`
```

Comment:
could you clean up the indentation of the code samples? the code should be indented one level from the indent level of the `dedent`, and the end backtick should be at the same indent as the `dedent`

i.e.


```suggestion
    dedent`
      it('pass', async () => {
        expect();
      })
    `,
    dedent`
```